### PR TITLE
Impressions and YearTotals update

### PR DESCRIPTION
**Pull request by damisul**
added migration to update impressions and year_totals records to reference authorities instead of people

### CODE REVIEW

File: db/schema.rb
Comment (abartov) [inline db/schema.rb:1]: This seems to have slipped in by mistake.

**Review by abartov — CHANGES_REQUESTED** [review]
exclude db/schema.rb with ingestible

File: db/schema.rb
Comment (damisul) [reply db/schema.rb:1]: Hm... actually not. I want to merge this into a master branch and master branch should not know anything about ingestible table for now

Comment (abartov) [reply db/schema.rb:1]: Oh, sorry! I instinctively misread the diff the other way around! :)
*I* am the one who mistakenly committed the ingestible. Sorry!

**Review by abartov — APPROVED** [review]

**Pull request merged**